apps/graphql/src/types/output/RouteStop.js

```diff
@@ -9,18 +9,19 @@ import type { GraphqlContextType } from '../../services/GraphQLContext';
 
 export default new GraphQLObjectType({
   name: 'RouteStop',
-  description: 'Departure or arrival for a segment',
+  description: 'Departure or arrival for a segment or sector',
   fields: {
     cityName: {
```

Comment:
I suggest to remove also `cityName`. The `Location` entity should have new property `city: LocationArea` from where name of the city can be resolved as `stop?.city?.name`

Reply:
Yes, that should probably be doable. I see the location query returns a field called city, we should be bale to get it from there.